packages/http-client-csharp/generator/Directory.Build.props

```diff
@@ -0,0 +1,55 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I know this is called this way for historical reasons but wondering if emitter vs generator really is the right naming

Reply:
what would you call it instead?

Reply:
I guess if you see `/emitter` as the parent then I guess its fine. Otherwise probably should structure this way
```
emitter:
   src/  Contains the emitter definition($onEmit, $lib) and orchestractor that calls the converter then generator
   converter?: TS Code that create the csharp code model
   generator: csharp code
```

Reply:
The concept of an Emitter in TypeSpec is not something that has an onEmit, but something which consumes the type graph to produce some useful output. The notion of a generator is an implementation detail.

Reply:
an implementation detail that needs a folder :-)

Reply:
Not opining on the structure, just clarifying the emitter terminology. Though I do like Timothee's last suggestion.

Reply:
preprocessor instead of converter works too